### PR TITLE
PIM: Remove existing role assignment

### PR DESCRIPTION
**Pull request by Garima-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/PIM/PIM.ps1 [hunk 1]
```diff
@@ -193,7 +203,10 @@ function Set-AzSKPIMConfiguration {
                 $pimconfig.InvokeFunction($pimconfig.Deactivate, @($SubscriptionId, $ResourceGroupName, $ResourceName, $RoleName))
             }
             elseif ($PSCmdlet.ParameterSetName -eq 'Assign') {				
-                $pimconfig.InvokeFunction($pimconfig.AssignExtendPIMRoleForUser, @($SubscriptionId, $ResourceGroupName, $ResourceName, $RoleName, $PrincipalNames, $DurationInDays, $false))
+                $pimconfig.InvokeFunction($pimconfig.AssignExtendPIMRoleForUser, @($SubscriptionId, $ResourceGroupName, $ResourceName, $RoleName, $PrincipalNames, $DurationInDays, $false, $false, $false))
```
Comment (SINIKI) [inline src/AzSK/PIM/PIM.ps1]: Force is hardcoded here. Is this expected?

File: src/AzSK/Framework/Core/PIM/PIMScript.ps1 [hunk 2]
```diff
@@ -457,7 +457,7 @@ class PIM: AzCommandBase {
     }
 
     #Assign a user to Eligible Role
-    hidden AssignExtendPIMRoleForUser($subscriptionId, $resourcegroupName, $resourceName, $roleName, $PrincipalName, $duration,$isExtnensionRequest) {
+    hidden AssignExtendPIMRoleForUser($subscriptionId, $resourcegroupName, $resourceName, $roleName, $PrincipalName, $duration,$isExtnensionRequest, $force, $isRemoveAssignmentReq) {
         $this.AcquireToken();
```
Comment (SINIKI) [inline src/AzSK/Framework/Core/PIM/PIMScript.ps1]: isExtnensionRequest -> Spelling correction -> isExtensionRequest 
isRemoveAssignmentReq -> We can use full name here -> 'isRemoveAssignmentRequest'

File: src/AzSK/PIM/PIM.ps1 (hunk 1 above)
Comment (Garima-msft) [reply src/AzSK/PIM/PIM.ps1]: Yes , as Force is only applicable for removerole command

File: src/AzSK/PIM/PIM.ps1 [hunk 3]
```diff
@@ -18,7 +18,11 @@ function Set-AzSKPIMConfiguration {
         [Alias("ar")]
 	    $AssignRole,
 
-		
+		[switch]
+        [Parameter(Mandatory = $false, ParameterSetName = "Remove", HelpMessage = "This switch is required to remove a PIM eligible role.")]
+        [Alias("rr")]
+        $RemoveRole,
```
Comment (v-zhgup) [inline src/AzSK/PIM/PIM.ps1]: RemovePIMAssignment may be a better name for the switch

File: src/AzSK/PIM/PIM.ps1 [hunk 4]
```diff
@@ -47,6 +51,7 @@ function Set-AzSKPIMConfiguration {
         [Parameter(Mandatory = $true, ParameterSetName = "Activate")]
         [Parameter(Mandatory = $true, ParameterSetName = "Deactivate")]
         [Parameter(Mandatory = $true, ParameterSetName = "Assign")]
+        [Parameter(Mandatory = $true, ParameterSetName = "Remove")]
```
Comment (v-zhgup) [inline src/AzSK/PIM/PIM.ps1]: Also the parametersetname is generic, if we can use removePIMAssignment

**Pull request merged**